woocommerce-gateway-paypal-express-checkout.php

```diff
@@ -1,10 +1,10 @@
 <?php
 /**
  * Plugin Name: WooCommerce PayPal Express Checkout Gateway
- * Plugin URI: https://www.woothemes.com/products/woocommerce-gateway-paypal-express-checkout/
+ * Plugin URI: https://woocommerce.com/products/woocommerce-gateway-paypal-express-checkout/
  * Description: A payment gateway for PayPal Express Checkout (https://www.paypal.com/us/webapps/mpp/express-checkout).
- * Version: 1.2.0
- * Author: Automattic
+ * Version: 1.2.1
```

Comment:
1.2.0 is not out from the oven. See list of releases in GH and latest version and https://wordpress.org/plugins/woocommerce-gateway-paypal-express-checkout/.